### PR TITLE
Create 2020-03-31-coreweekly-week-13.md

### PR DESCRIPTION
**Pull request by ttoine**

### CODE REVIEW

File: news/_posts/core-weekly/2020-03-31-coreweekly-week-13.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+### Tests
+* [#18309](https://github.com/PrestaShop/PrestaShop/pull/18309): Change test fixtures that need to be in the future, by [@jolelievre](https://github.com/jolelievre)
+
+
+## Code changes in modules, themes & tools
+
+
+### Changes in developer documentation
+* [#495](https://github.com/PrestaShop/docs/pull/495): Change node version as we no support 12.x LTS, by [@NeOMakinG](https://github.com/NeOMakinG)
```
Comment (Progi1984) [inline news/_posts/core-weekly/2020-03-31-coreweekly-week-13.md]:
```suggestion
* [#495](https://github.com/PrestaShop/docs/pull/495): Change node version as we now support 12.x LTS, by [@NeOMakinG](https://github.com/NeOMakinG)
```

Comment (ttoine) [reply news/_posts/core-weekly/2020-03-31-coreweekly-week-13.md]: then could you please change also the title of the relative pull request.

Comment (Progi1984) [reply news/_posts/core-weekly/2020-03-31-coreweekly-week-13.md]: Done !

Comment (ttoine) [reply news/_posts/core-weekly/2020-03-31-coreweekly-week-13.md]: thanks !

**Review by Progi1984 — APPROVED** [review]

File: news/_posts/core-weekly/2020-03-31-coreweekly-week-13.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+
+### Faceted search module
+* [#166](https://github.com/PrestaShop/ps_facetedsearch/pull/166): remove trailing space, by [@matthieu-rolland](https://github.com/matthieu-rolland)
+* [#165](https://github.com/PrestaShop/ps_facetedsearch/pull/165): Release 3.5.0, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#164](https://github.com/PrestaShop/ps_facetedsearch/pull/164): Bump to 3.5.0, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#163](https://github.com/PrestaShop/ps_facetedsearch/pull/163): Update npm packages, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#154](https://github.com/PrestaShop/ps_facetedsearch/pull/154): Sort products: InStock, available with qty 0, OutOfStock. Thank you [@gadnis](https://github.com/gadnis)
+
+
+### ps_livetranslation
```
Comment (ttoine) [inline news/_posts/core-weekly/2020-03-31-coreweekly-week-13.md]:
```suggestion
### Live translation
```

File: news/_posts/core-weekly/2020-03-31-coreweekly-week-13.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+
+### Contact informations module
+* [#26](https://github.com/PrestaShop/ps_contactinfo/pull/26): Release v3.3.0, by [@jolelievre](https://github.com/jolelievre)
+
+
+### Email subscription module
+* [#51](https://github.com/PrestaShop/ps_emailsubscription/pull/51): Do not send voucher when customer signs up if he did not register for newsletter , by [@matthieu-rolland](https://github.com/matthieu-rolland)
+
+
+### ps_themecusto
```
Comment (ttoine) [inline news/_posts/core-weekly/2020-03-31-coreweekly-week-13.md]:
```suggestion
### Theme customization
```

File: news/_posts/core-weekly/2020-03-31-coreweekly-week-13.md [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+### Prestashop UI Kit
+* [#90](https://github.com/PrestaShop/prestashop-ui-kit/pull/90): Update webpack from 3.22 to 4.42, by [@NeOMakinG](https://github.com/NeOMakinG)
+* [#89](https://github.com/PrestaShop/prestashop-ui-kit/pull/89): Update bootstrap version from 4.0 to 4.4.1, by [@NeOMakinG](https://github.com/NeOMakinG)
+* [#88](https://github.com/PrestaShop/prestashop-ui-kit/pull/88): Do improvements corrections of UIKit evolutions, by [@NeOMakinG](https://github.com/NeOMakinG)
+* [#87](https://github.com/PrestaShop/prestashop-ui-kit/pull/87): Adding error to group input and adjusting some things, by [@NeOMakinG](https://github.com/NeOMakinG)
+* [#86](https://github.com/PrestaShop/prestashop-ui-kit/pull/86): Adding PSInputNumber component, by [@NeOMakinG](https://github.com/NeOMakinG)
+* [#79](https://github.com/PrestaShop/prestashop-ui-kit/pull/79): Refactor yes/no toggle component, by [@NeOMakinG](https://github.com/NeOMakinG)
+
+
+### ps_crossselling
```
Comment (ttoine) [inline news/_posts/core-weekly/2020-03-31-coreweekly-week-13.md]:
```suggestion
### Cross-selling
```

File: news/_posts/core-weekly/2020-03-31-coreweekly-week-13.md [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+* [#7](https://github.com/PrestaShop/ps_crossselling/pull/7): Cross selling products not displayed on displayShoppingCartFooter hook. Thank you [@Nukium](https://github.com/Nukium)
+
+
+### statslive
```
Comment (ttoine) [inline news/_posts/core-weekly/2020-03-31-coreweekly-week-13.md]:
```suggestion
### Visitors online
```

File: news/_posts/core-weekly/2020-03-31-coreweekly-week-13.md [hunk 6]
```diff
@@ -0,0 +1,194 @@
...
+* [#8](https://github.com/PrestaShop/statslive/pull/8): Release 2.1.0, by [@PierreRambaud](https://github.com/PierreRambaud)
+
+
+### psgdpr
```
Comment (ttoine) [inline news/_posts/core-weekly/2020-03-31-coreweekly-week-13.md]:
```suggestion
### psgdpr
```

File: news/_posts/core-weekly/2020-03-31-coreweekly-week-13.md [hunk 7]
```diff
@@ -0,0 +1,194 @@
...
+
+## Code changes in modules, themes & tools
+
+
+### Changes in developer documentation
+* [#495](https://github.com/PrestaShop/docs/pull/495): Change node version as we now support 12.x LTS, by [@NeOMakinG](https://github.com/NeOMakinG)
+* [#494](https://github.com/PrestaShop/docs/pull/494): Fix typo mistake. Thank you [@gadnis](https://github.com/gadnis)
+
+
+### open-source
```
Comment (ttoine) [inline news/_posts/core-weekly/2020-03-31-coreweekly-week-13.md]:
```suggestion
### The PrestaShop open source project
```

**Review by ttoine — COMMENTED** [review]
I also created https://github.com/PrestaShop/core-weekly-generator/pull/28 to add missing modules to the list.

File: news/_posts/core-weekly/2020-03-31-coreweekly-week-13.md (hunk 6 above)
Comment (ttoine) [reply news/_posts/core-weekly/2020-03-31-coreweekly-week-13.md]:
```suggestion
### PSGDPR
```

File: news/_posts/core-weekly/2020-03-31-coreweekly-week-13.md [hunk 8]
```diff
@@ -0,0 +1,194 @@
...
+* [#18250](https://github.com/PrestaShop/PrestaShop/pull/18250): Functional tests - Add test 'Filter orders', by [@boubkerbribri](https://github.com/boubkerbribri)
+* [#18242](https://github.com/PrestaShop/PrestaShop/pull/18242): Functional tests - Add test 'Help card' for orders page, by [@boubkerbribri](https://github.com/boubkerbribri)
+* [#18231](https://github.com/PrestaShop/PrestaShop/pull/18231): Functional tests - Fix default timeout for waiting for selector, by [@boubkerbribri](https://github.com/boubkerbribri)
+* [#18197](https://github.com/PrestaShop/PrestaShop/pull/18197): Functional tests - Order settings enable/disable reordering option. Thank you [@nesrineabdmouleh](https://github.com/nesrineabdmouleh)
+* [#18161](https://github.com/PrestaShop/PrestaShop/pull/18161): Functional tests - Add test Enable and disable B2B mode, by [@boubkerbribri](https://github.com/boubkerbribri)
+* [#17965](https://github.com/PrestaShop/PrestaShop/pull/17965): Add HTML IDs and classes to View Customer BO page, by [@matks](https://github.com/matks)
+
+
+## Code changes in the '1.7.6.x' branch
```
Comment (matks) [inline news/_posts/core-weekly/2020-03-31-coreweekly-week-13.md]:
```suggestion
## Code changes in the '1.7.6.x' branch (for v1.7.6.5)
```

File: news/_posts/core-weekly/2020-03-31-coreweekly-week-13.md [hunk 9]
```diff
@@ -0,0 +1,194 @@
...
+### Tests
+* [#18285](https://github.com/PrestaShop/PrestaShop/pull/18285): Fix case in path. Thank you [@mvorisek](https://github.com/mvorisek)
+
+
+### Merge
+* [#18331](https://github.com/PrestaShop/PrestaShop/pull/18331): Merge 1.7.7.x into develop - 26/03/2020, by [@matks](https://github.com/matks)
+* [#18244](https://github.com/PrestaShop/PrestaShop/pull/18244): Merge 1.7.7.x into develop - 20/03/2020, by [@matks](https://github.com/matks)
+
+
+## Code changes in the '1.7.7.x' branch
```
Comment (matks) [inline news/_posts/core-weekly/2020-03-31-coreweekly-week-13.md]:
```suggestion
## Code changes in the '1.7.7.x' branch (for v1.7.7.0)
```

File: news/_posts/core-weekly/2020-03-31-coreweekly-week-13.md [hunk 10]
```diff
@@ -0,0 +1,194 @@
...
+* [#51](https://github.com/PrestaShop/ps_emailsubscription/pull/51): Do not send voucher when customer signs up if he did not register for newsletter , by [@matthieu-rolland](https://github.com/matthieu-rolland)
+
+
+### Theme customization
+* [#23](https://github.com/PrestaShop/ps_themecusto/pull/23): Bump version 1.1.1, by [@jolelievre](https://github.com/jolelievre)
+* [#22](https://github.com/PrestaShop/ps_themecusto/pull/22): Update references of strings in translations, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+
+
+### Prestashop UI Kit
+* [#90](https://github.com/PrestaShop/prestashop-ui-kit/pull/90): Update webpack from 3.22 to 4.42, by [@NeOMakinG](https://github.com/NeOMakinG)
```
Comment (matks) [inline news/_posts/core-weekly/2020-03-31-coreweekly-week-13.md:158]: FYI this PR allows us to support Node.js 11 and 12 (previously we only supported Node.js v8 to v10)

**Comment by matks** [comment]
FYI I added suggestions (not mandatory)
It seems there was 3 module releases last week: Faceted Search, Contact Info and Stats Live. Might be worth mentioning

**Comment by matks** [comment]
(sorry for bad close, I wanted to click on "comment" and I misclicked the "close and comment" button 😓 )

**Comment by matks** [comment]
@ttoine Approved, feel free to merge

**Review by matks — APPROVED** [review]

File: news/_posts/core-weekly/2020-03-31-coreweekly-week-13.md [hunk 11]
```diff
@@ -0,0 +1,194 @@
...
+image: /assets/images/2017/04/core_weekly_banner.jpg
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 23rd to Sunday 29th of March 2020.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
```
Comment (ttoine) [inline news/_posts/core-weekly/2020-03-31-coreweekly-week-13.md:17]:
```suggestion
## General Messages

Last week, new versions of Faceted Search, Contact Info and Stats Live have been released. Don't forget to update them!
```

**Pull request merged**